src/app/ui/tooltip.js

```diff
@@ -0,0 +1,52 @@
+var yo = require('yo-yo')
+// -------------- styling ----------------------
+var csjs = require('csjs-inject')
+var remixLib = require('remix-lib')
+var styleGuide = remixLib.ui.styleGuide
+var styles = styleGuide()
+
+var css = csjs`
+  .tooltip {
+    visibility: hidden;
+    width: 100px;
+    background-color: ${styles.remix.tooltip_CopyToClipboard_BackgroundColor};
+    color: ${styles.remix.tooltip_CopyToClipboard_Color};
+    font-weight: bold;
+    font-size: 12px;
+    text-transform: initial;
+    text-align: center;
+    border-radius: 3px;
+    padding: 5px 0;
+    position: absolute;
+    z-index: 1;
+    bottom: 160%;
+    left: 100%;
+    margin-left: -60px;
+    opacity: 0;
+    transition: opacity 1s;
+  }
+  .tooltip::after {
+      content: "";
+      position: absolute;
+      top: 100%;
+      left: 50%;
+      margin-left: -5px;
+      border-width: 5px;
+      border-style: solid;
+      border-color: ${styles.remix.tooltip_CopyToClipboard_BackgroundColor} transparent transparent transparent;
+  }
+`
+
+module.exports = function addTooltip (event) {
```

Comment:
can this tool tip be also used when creating a transaction? (for the next PR)

Reply:
@yann300 Sure, we can make a module and reuse it on many places. If you think you would like this, please create an issue and assign me and I will work on it after the IDE.

Reply:
this one https://github.com/ethereum/browser-solidity/issues/911

Reply:
thanks!